persistence-jdbc/core/src/main/resources/reference.conf

```diff
@@ -10,8 +10,17 @@ jdbc-defaults.slick {
   jndiName = "DefaultDS"
 }
 
-# Configure the default database to be bound to JNDI
-db.default.jndiName=DefaultDS
+
+db.default {
+  # Configure the default database to be bound to JNDI
+  jndiName=DefaultDS
+
+  hikaricp {
+    minimumIdle = 20
+    maximumPoolSize = 100
+  }
```

Comment:
While we usually recommend HikariCP I think it is also possible to mix in BoneCP using Play's [BoneCPComponenets](https://www.playframework.com/documentation/2.6.x/api/scala/index.html#play.api.db.BoneCPComponents).

We stir people to Hikari, even [in the docs](https://www.lagomframework.com/documentation/1.4.x/scala/ScalaComponents.html) though.

Reply:
Also, I would add a message on why the defaults are `20` and `100` and what impact it could have if changed.

Reply:
You are right. 
We need to clearly communicate over that and document. There is another issue about the documentation as well. I suggest we keep that on hold for a while because we don't even know what will be the final solution.